src/test/integration/java/com/commercetools/payment/BasePaymentIT.java

```diff
@@ -69,16 +71,29 @@
     protected SphereClient sphereClient;
     protected CtpFacade ctpFacade;
 
-    public void setUp() throws Exception {
+    /**
+     * Cleanup orders, carts and payments storages.
+     */
+    public void setupBeforeAll() {
+        initTenantConfigs();
+        cleanOrdersCartsPayments(sphereClient);
+    }
+
+    @Before
+    public void setUp() {
+        initTenantConfigs();
+    }
+
+    /**
+     * Instantiate request/tenant specific values (configs, facades, factories, clients) before each test.
+     */
+    protected void initTenantConfigs() {
```

Comment:
why protected? is `private` not enough?

Reply:
not enough for what? one could override it if necessary for custom tests